README.md

```diff
@@ -72,6 +72,18 @@ GET example.com/api/v1/blogs/1
 GET example.com/api/v1/tech/blogs/1
 ```
 
+To sort your results, use **sort_by** as a query parameter. The values of **sort_by** parameter is a comma-separated string in the format of `sort_field:sort_order`. If `sort_order` is not specified, the default sort order `asc` will be used
+```
+# Sort results by position - from the first to the last (ascending)
+GET example.com/api/v1/blogs/?sort_by=position
+
+# Sort results by created_at - from the newest to the oldest (descending)
+GET example.com/api/v1/blogs/?sort_by=created_at:desc
+
+# Sort results by position - from the first to the last. If there are records with the same position, sort by created_at - from the newest to the oldest
+GET example.com/api/v1/blogs/?sort_by=position,created_at:desc
```

Comment:
For multi-value param, it's better to use something close to standard multi-value query param:
For single value:
`sort=position` or with order `sort=position:desc`
For multi-value:
`sort[]=position&sort[]=created_at` or with order `sort[]=position:asc&sort[]=created_at:desc`
This way, users can specify different sorting order per field.

Reply:
@sandyandi I've changed the sorting to use multi-value query param. Please help to review.